index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
+            the audio <a>rendering thread</a>. It is defined in an
+            <a>AudioWorkletGlobalScope</a> and the definition of such class
+            manifests the actual audio processing mechanism of an
+            <a>AudioWorkletNode</a>. <a>AudioWorkletProcessor</a> can only be
+            instantiated by the construction of an <a>AudioWorkletNode</a>
+            instance.
+          </p>
+          <dl title="interface AudioWorkletProcessor" class="idl">
+            <dt>
+              readonly attribute static sequence&lt;AudioParamDescriptor&gt;
+            </dt>
+            <dd>
+              <p>
+                This array contains <a>AudioParamDescriptors</a>, which are
+                descriptors for each controllable parameter on an
+                <a>AudioNode</a>. This static property is referred in the
+                construction of an <a>AudioWorkletNode</a> to create instances
+                of <a>AudioParam</a> in the node.
+              </p>
+            </dd>
+            <dt>
+              readonly attribute AudioContextInfo contextInfo
+            </dt>
+            <dd>
+              <p>
+                This getter provides an object contains the information of the
+                associated <a>AudioContext</a> such as the sample rate and the
+                current playback time when queried.  
+              </p>
+            </dd>
+            <dt>
+              attribute EventHandler onmessage
+            </dt>
+            <dd>
+              <p>
+                The onmessage handler is called whenever the
+                <a>AudioWorkletNode</a> posts a node message back to the
+                corresponding processor object on the audio <a>rendering
+                thread</a>.  
+              </p>
+            </dd>
+            <dt>
+              void process()
+            </dt>
+            <dd>
+              <p>
+                This method gets executed isochronously by the audio rendering
+                thread along with the rest of an audio graph while the
+                associated <a>AudioWorkletNode</a> is connected.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  Float32Array[][] inputs
+                </dt>
+                <dd>
+                  Represents incoming data from inputs of the node. Each input
+                  can contain multiple channels.
+                </dd>
+                <dt>
+                  Float32Array[][] outputs
+                </dt>
+                <dd>
+                  Represents outgoing data from outputs of the node. Each output
+                  can contain multiple channels.
+                </dd>
+                <dt>
+                  Object parameters
+                </dt>
+                <dd>
+                  Represents an object contains multiple arrays of
+                  <a>AudioParam</a> data with the associated name as a key.
```

Comment:
If parameters is described as containing arrays, then should parameters be declared to be an array?  Exactly what kind of object is parameters?  Is it a dictionary?